src/librustc_driver/driver.rs

```diff
@@ -61,6 +61,12 @@ use syntax::visit;
 use syntax;
 use syntax_ext;
 
+pub struct CrateMap {
```

Comment:
I wonder if this name could describe what is being mapped - `Crate` is very generic. Maybe `ResolutionMap` or `NameMap` or something? Or even dump the word `Map`.